.github/workflows/CD.yml

```diff
@@ -121,10 +129,23 @@ jobs:
           echo ${{ steps.set-matrix.outputs.ec2-matrix-1 }}
           echo ${{ steps.set-matrix.outputs.ec2-matrix-2 }}
           echo ${{ steps.set-matrix.outputs.ec2-matrix-3 }}
-          
-  release-to-s3:
+
+  get-release-region:
```

Comment:
Yes. Because matrix does not support environment variables currently (tested it out with my branch) and only support hard code one, therefore, I use similar idea with get-test-suit it the CI to do the dynamic variable.

Reply:
Didn't know that. It's so strange that it doesn't work.

Reply:
Same.I got the startup failure many times so.